__tests__/app.test.js

```diff
@@ -7,6 +7,17 @@ const app = require('../app');
 beforeEach(() => seed(data));
 afterAll(() => db.end());
 
+describe('ALL non-existent paths', () => {
```

Comment:
Nice to see this broken out to its own test block since it doesn't directly relate to any of the other endpoints